core/string/ustring.cpp

```diff
@@ -3507,6 +3507,27 @@ char32_t String::unicode_at(int p_idx) const {
 	return operator[](p_idx);
 }
 
+String String::indent(const String &p_indent) const {
```

Comment:
```suggestion
String String::indent(const String &p_prefix) const {
```
to match the exposed name.